WikiWatershed/model-my-watershed
# Add observation popups

This is very unstyled because @jfrankl is working on an ultimate design/layout style.  This merely get's the mechanics in place, the ultimate flow will be added later.

* Observation Popups have basic details and a listing of all variables measured at that station, with most recent values
* Clicking a variable will open a detail view modal with a timeseries plot of 2 months of data

Connects #1008 

**This is getting merged into the `feature/observation` branch on `origin` to avoid being deployed until it is ready.**

To Test:
1.  Add an observation layer to the map via the Layer Selector (USGS adds some points in Philly)
1. Click on an asset marker to see the popup.
1. Click on any `li` measurement item
1. See a modal appear plotting a few months worth of observation data for that asset and measurement
